view/template/developer/_quickstartCredits.php

```diff
@@ -19,7 +19,16 @@
   "name": "electricsheep",
   "file_path": "\\home\kauffj\\Desktop\\electric-sheep.mp4",
   "bid": 1,
-  "metadata":  { } <span class="code-bash__comment">//this should match the metadata returned by resolve_name </span>
+  "metadata":  {
+    "description": "Some interesting content",
+    "title": "my_interesting_content",
+    "author": "Video shot by me@example.com",
+    "language": "en",
+    "license": "LBRY Inc",
+    "ver": "0.1",
+    "nsfw": false,
+    "sources": {}
```

Comment:
According to the schema it's still required, but maybe it's filled in automatically when publishing files?